content/deploy/cluster-sizing.md

```diff
@@ -1,29 +1,77 @@
+For the Rhize Agent, the minimum recommended specifications are as follows:
+
+| Property              | Value |
+|-----------------------|-------|
+| CPU Speed (GHz)       | 2.8   |
+| vCPU per Node         | 2     |
+| Memory per node (GiB) | 1     |
+| Persisted volumes     | 12    |
+
+## Service-level recommendations
+
+The following table lists the **minimum** recommended specfications for the main services.
+Services with stateful PV have a persistent volume per pod.
+
+
+| Service                | Pods for HA (replica count) | vCPU per Pod | Memory Per Pod | Stateful PV | DiskSize (GiB) | Comments                                    |
+|------------------------|-----------------------------|--------------|----------------|-------------|----------------|---------------------------------------------|
+| `baas-alpha`           | 3                           | 8            | 16 (at least)  | Yes         | 750            | High throughput and IOPS                    |
+| `baas-zero`            | 3                           | 2            | 2              | Yes         | 350            | High throughput and IOPS                    |
+| `libre-core`           | 3                           | 1            | 2              | No          | N/A            |                                             |
+| `bpmn-engine`          | 3                           | 1            | 2              | No          | N/A            |                                             |
+| `nats`                 | 3                           | 1            | 2              | Yes         | 100GB          | High IOPS                                   |
+| `nats-box`             | 1                           | 0.25         |                | No          | N/A            |                                             |
+| `libre-audit`          | 2                           |              |                |             | N/A            |                                             |
+| `libre-audit-postgres` | 2                           |              |                | Yes         | 250            | runs in pod with `libre-audit`              |
+| `libre-ui`             | 3                           | 0.25         | 0.25           |             | N/A            |                                             |
+| `keycloak`             | 2                           | 1            | 2              |             | N/A            |                                             |
+| `keycloak-postgres`    | 2                           | 1            | 2              |             | 200            | runs in pod with `keycloak`                 |
+| `router`               | 2                           | 1            | 2              | Yes         | <1             | Requires volume to compose supergraph       |
+| `grafana`*             | 3                           | 0.5          | 2              | NO          | 20-50          | Storage can be in host or in object bucket. |
```

Comment:
Grafana can be just 2 pods for HA.

The "NO" in stateful PV is capitalized whilst it is "No" in the others. Can you change it to "No" to be consistent.